Jenkinsfile

```diff
@@ -0,0 +1,43 @@
+pipeline {
+  agent {
+    node {
+      label 'mac-os'
+    }
+    
+  }
+  stages {
+    stage('Build and test') {
+      when {
+        expression {
+          result = sh (script: "git log -1 | grep '.*\\[ci skip\\].*'", returnStatus: true) // Check if commit message contains skip ci label
+          result != 0 // Evaluate the result
+        }
+        
+      }
+      environment {
+        GIT = credentials('65325e52-5ec0-46a7-a937-f81f545f3c1b')
+      }
+      steps {
+        deleteDir()
+        checkout scm
+        dir(path: 'Yona') {
+          sh '/usr/local/bin/pod install'
+          sh 'xcrun agvtool new-marketing-version 1.1'
+          sh 'xcrun agvtool next-version -all'
+          sh 'xcodebuild -allowProvisioningUpdates -workspace Yona.xcworkspace -configuration Debug -scheme Yona archive -archivePath ./BuildOutput/Yona-Debug-$BUILD_NUMBER.xcarchive'
+          sh 'xcodebuild -exportArchive -archivePath ./BuildOutput/Yona-Debug-$BUILD_NUMBER.xcarchive -exportPath ./BuildOutput/Yona-Debug-$BUILD_NUMBER.ipa -exportOptionsPlist ./BuildOutput/ExportOptionsDebug.plist'
+          sh 'xcodebuild -allowProvisioningUpdates -workspace Yona.xcworkspace -configuration Release -scheme Yona archive -archivePath ./BuildOutput/Yona-Release-$BUILD_NUMBER.xcarchive'
+          sh 'xcodebuild -exportArchive -archivePath ./BuildOutput/Yona-Release-$BUILD_NUMBER.xcarchive -exportPath ./BuildOutput/Yona-Release-$BUILD_NUMBER.ipa -exportOptionsPlist ./BuildOutput/ExportOptionsRelease.plist'
+          sh 'git checkout $BRANCH_NAME'
+          sh 'git add -u'
+          sh 'git commit -m "Updated versionCode for build $BUILD_NUMBER [ci skip]"'
+          sh 'git push https://${GIT_USR}:${GIT_PSW}@github.com/yonadev/yona-app-ios.git'
```

Comment:
We can leave this first push out, because we can push the commit and the tag together two lines below.

Reply:
Don't think so. The ``--tags`` option pushes the tags only.